utils.groovy

```diff
@@ -427,6 +427,41 @@ def build_artifacts(pipecfg, stream, basearch) {
     // First get the list of artifacts to build from the config
     def artifacts = get_artifacts_to_build(pipecfg, stream, basearch)
 
+    // If `cosa osbuild` is supported then let's build what we can using OSBuild
+    if (shwrapRc("cosa shell -- test -e /usr/lib/coreos-assembler/cmd-osbuild") == 0) {
+        // Determine which platforms are OSBuild experimental versus
+        // stable (i.e. the default is to use OSBuild for them).
+        def experimental = shwrapCapture("cosa osbuild --supported-platforms").tokenize()
+        def stable = shwrapCapture('''
+            cosa shell -- bash -c '
+                for buildextend in /usr/lib/coreos-assembler/cmd-buildextend-*; do
+                    if [ "$(readlink -f ${buildextend})" == "/usr/lib/coreos-assembler/cmd-osbuild" ]; then
+                        echo "${buildextend:42}"
```

Comment:
That 42 is worth a comment. Or alternatively, I sometimes make it more self-documenting by putting the prefix into a variable, and then you can use it in the iteration as well as here via e.g. `${#prefix}`.

Reply:
to prevent re-testing in CI I'll just add a comment.